bootstrap.sh

```diff
@@ -6,6 +6,7 @@ cd /vagrant
 pipenv sync --dev
 pipenv install Pillow
 pipenv run python manage.py migrate
+pipenv install --dev pytest
```

Comment:
if you used the following:
`pipenv install --dev pytest`
`pipenv install --dev pytest-django`
then pytest and pytest django are already in the dev tools and are installed by `pipenv sync --dev`
so line 9 is redundant here, I think

Reply:
Thanks!
Seems it was not needed.

Not sure i understand though - if we need to manually install it, why we don't need to add it to the bootstrap file?

i mean, why do we need to run on our VM:
`pipenv install --dev pytest`
`pipenv install --dev pytest-django`

but after the Pipefiles are updated - we dont need to run it any more?

Reply:
because you update the pipfile once - it's enough. It lives there now. Try destroying your VM and recreating it and see that you can use this, as is.
If I understood correctly, pipfile keeps the developer tools for us and automatically downloads them when called to `sync --dev`.
So as you've added it while using the VM and it lives in the pipfile now, there's no point in denoting this in the bootstrap.sh file